routes/index.js

```diff
@@ -251,21 +251,16 @@ var routes = {
     var transaction = request.payload || {};
     var frequency = transaction.frequency || "";
     var currency = transaction.currency;
-    var appName = transaction.appName;
     var amount = amountModifier.paypal(transaction.amount, currency);
-    var returnUrl = request.server.info.uri + '/api/paypal-redirect/' + frequency + '/' + transaction.locale + '/';
-    if (appName === "thunderbird") {
-      returnUrl += "thunderbird/";
-    } else {
-      returnUrl += "mozillafoundation/";
```

Comment:
Instead of building this now, and passing it around. I decided to just build it when we need it.